### DISCUSSION TITLE
File not showing in FileUpload

### DISCUSSION THREAD

**Discussion by kcanyildirim** [filamentphp/filament#6930] [Help]
Hello, the data I uploaded does not appear in FileUpload. Where could the problem be? The file is being loaded from the application form, not from Filament.

`
   FileUpload::make('cv')
                            ->label('CV')
                            ->disk('public')
                            ->directory('uploads')
                            ->visibility('public'),
`

**Comment by pboivin** [comment]
What are you doing with the uploaded file path, are you saving it to the DB? Can you share more code?

**Reply by kcanyildirim** [reply]
Yes, I'm saving it to the database, I saved it as path and url, I tried, but neither of them shows the preview.

**Reply by danharrin** [reply]
https://filamentphp.com/tricks/file-upload-previews-not-loading
